pom.xml

```diff
@@ -160,7 +160,7 @@
     <grpc.version>1.33.0</grpc.version>
     <jackson-databind.version>2.9.10.7</jackson-databind.version>
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
-    <hapi-proto.version>0.13.0-alpha.3-SNAPSHOT</hapi-proto.version>
+    <hapi-proto.version>0.13.0-alpha.3-new-address-book-SNAPSHOT</hapi-proto.version>
```

Comment:
This version name should be changed `0.13.0-rc1-SNAPSHOT` ?

Reply:
Publishing `0.13.0-rc.1` now, will open new PR once it's available in Maven Central. 👍